include/kframework/gas.md

```diff
@@ -0,0 +1,50 @@
+KEVM Gas
+========
+
+Here is a representation of `Gas`. This sort is used by `<gas>`, `<callGas>`, and `<gasUsed>` cells of the EVM configuration. As is in this file, `Gas` is a super-sort of `Int` with no differing behaviour. However the distinction at this level allows for an extension of infinite gas later, see `infinite-gas.md`.
+
+```k
+module GAS-SYNTAX
+    imports INT-SYNTAX
+    
+    syntax Gas ::= Int
+    syntax Int ::= "gas2Int" "(" Gas ")" [function, total]
+    
+    syntax Gas ::= "minGas" "(" Gas "," Gas ")" [function, total]
+                 > left:
+                   Gas "*Gas" Gas [function, left, total]
+                 | Gas "/Gas" Gas [function, left]
+                 > left:
+                   Gas "+Gas" Gas [function, left, total]
+                 | Gas "-Gas" Gas [function, left, total]
+
+    syntax Bool ::= Gas  "<Gas" Gas [function, left, total]
+                  | Gas "<=Gas" Gas [function, left, total]
```

Comment:
```suggestion
    syntax Gas ::= "minGas" "(" Gas "," Gas ")" [function, total]
                 > left:
                   Gas "*Gas" Gas [function, total]
                 | Gas "/Gas" Gas [function]
                 > left:
                   Gas "+Gas" Gas [function, total]
                 | Gas "-Gas" Gas [function, total]

    syntax Bool ::= Gas  "<Gas" Gas [function, total]
                  | Gas "<=Gas" Gas [function, total]
```

In the case of the arithmetic operators, these `left` annotations are redundant with the `left: ...` component.

In the case of the boolean operators, they don't make sense because they aren't associative operator.s